telegram/_message.py

```diff
@@ -1593,7 +1604,7 @@ async def reply_text(
             allow_sending_without_reply=allow_sending_without_reply,
             entities=entities,
             protect_content=protect_content,
-            message_thread_id=message_thread_id,
+            message_thread_id=message_thread_id or self.message_thread_id,
```

Comment:
```suggestion
            message_thread_id=message_thread_id or (self.message_thread_id if chat_id in {self.chat_id, self.username} else None),
```

Reply:
Makes much sense.
Only in my branch, in master if user passed message_thread_id by themselves but the PS. cleared it :)